web/src/main/scala/org/hyperscala/io/bootstrap/Specification.scala

```diff
@@ -0,0 +1,578 @@
+
+    DefComponent("DropdownToggle", None,
+      Some("a"),
+      Set.empty,
+      matchOrFail("dropdown-toggle")),
+
+    DefComponent("DropdownHeader", None,
+      Some("li"),
+      Set.empty,
+      matchOrFail("dropdown-header")),
+
+    DefComponent("Panel", None,
+      Some("div"),
+      Set.empty,
+      matchOrFail("panel"),
+      Property("panelType",
+        Value.Set("PanelType",
+          Value.Option("Default", "panel-default"),
+          Value.Option("Primary", "panel-primary"),
+          Value.Option("Success", "panel-success"),
+          Value.Option("Info", "panel-info"),
+          Value.Option("Warning", "panel-warning"),
+          Value.Option("Danger", "panel-danger")
+        )
+      )
+    ),
+
+    DefComponent("ProgressBar", None,
+      Some("div"),
+      Set.empty,
+      matchOrFail("progress"),
+      Property("striped", Value.Boolean("progress-striped")),
+      Property("active", Value.Boolean("progress-active"))
+    ),
+
+    DefComponent("Row", None, Some("div"), Set.empty, matchOrFail("row")),
+
+    DefComponent("Table", None,
+      Some("table"),
+      Set.empty,
+      matchOrFail("table"),
+      Property("striped", Value.Boolean("table-striped")),
+      Property("bordered", Value.Boolean("table-bordered")),
+      Property("hover", Value.Boolean("table-hover")),
+      Property("condensed", Value.Boolean("table-condensed"))
+    ),
+
+    DefComponent("TableRow", Some("Table"),
+      Some("tr"),
+      Set.empty,
+      matchOrFail(),
+      Property("rowStyle",
+        Value.Set("RowStyle",
+          Value.Option("Default", "default"),
+          Value.Option("Active", "active"),
+          Value.Option("Success", "success"),
+          Value.Option("Info", "info"),
+          Value.Option("Warning", "warning"),
+          Value.Option("Danger", "danger")
+        )
+      )
+    ),
+
+    DefComponent("Tabs", None,
+      Some("ul"),
+      Set.empty,
+      matchOrFail("nav"),
+      Property("tabs", Value.Boolean("nav-tabs")),
+      Property("pills", Value.Boolean("nav-pills")),
+      Property("stacked", Value.Boolean("nav-stacked")),
+      Property("justified", Value.Boolean("nav-justified"))
+    ),
+
+    DefComponent("TabEntry", Some("Tabs"),
+      Some("li"),
+      Set.empty,
+      matchOrFail(),
+      Property("active", Value.Boolean("active")),
+      Property("disabled", Value.Boolean("disabled"))
+    ),
+
+    DefComponent("Well", None,
+      Some("div"),
+      Set.empty,
+      matchOrFail("well"),
+      Property("wellType",
+        Value.Set("WellType",
+          Value.Option("Default", "well-default"),
+          Value.Option("Small", "well-sm"),
+          Value.Option("Larger", "well-lg")
+        )
+      )
+    ),
+
+    DefComponent("ListItem", None,
+      Some("li"),
+      Set.empty,
+      matchOrFail(),
+      Property("active", Value.Boolean("active")))
+  )
+
+  // TODO How can we refer to org.hyperscala.bootstrap.component.Glyphicon?
```

Comment:
``bootstrap`` is a different project and I didn't manage to modify the build file, so that I could access ``org.hyperscala.bootstrap.component.Glyphicon``. There is a lot of duplication here. It would be nice to just refer to the auto-generated glyphicon list.